DEVELOPERNOTES.rst

```diff
@@ -77,6 +80,57 @@ Solr changes not reflected in search results? ``solrconfig.xml`` must be
 updated in Solr's main directory: ``solr/server/solr/[CORE]/conf/solrconfig.xml``
 
 
+Getting full-text data
+----------------------
+
+To get a full copy of all PPA data for development, rsync from the
+data directory on one of the staging VMs.  The majority of the content is
+stored on an NFS mount point under `/mnt/nfs/cdh/prosody/data/`:
+- `ht_text_pd`: HathiTrust pairtree data for PPA volumes
+- `eebo_tcp`: XML and MARC records for EEBO-TCP volumes included in PPA
+- `marc`: MARC records for Gale/ECCO content, in a pairtree format used for importing Gale content
+
+Local OCR for Gale/ECCO content is on a TigerData NFS mount point:
+- `/mnt/tigerdata/cdh/prosody/ppa-ocr/Gale-by-vol`
```

Comment:
`/mnt/tigerdata/cdh` appears to be empty?

<img width="373" alt="Screenshot 2025-07-01 at 1 02 07 PM" src="https://github.com/user-attachments/assets/a104d720-31e4-4563-a284-497317d035da" />

Reply:
It looks like it's broken on cdh-test-prosody1 at the moment, thanks for letting me know. You can use cdh-test-prosody2 as a fallback for now